subsys/ipc/ipc_service/backends/ipc_rpmsg_static_vrings.c

```diff
@@ -495,3 +495,25 @@ static int backend_init(const struct device *instance)
 			 &backend_ops);
 
 DT_INST_FOREACH_STATUS_OKAY(DEFINE_BACKEND_DEVICE)
+
+#define BACKEND_CONFIG_INIT(n) config[n] = &backend_config_##n;
+
+#if CONFIG_IPC_SERVICE_BACKEND_RPMSG_SHMEM_RESET
```

Comment:
`#ifdef` or `#if defined()`

Reply:
This will always be defined but depending on its value 0 or 1 it is considered enabled or disabled. That's why I used #if macro

Reply:
boolean symbols not set or set to `n` do not end up in `autoconf.h` so they are not actually defined (they are not set to 0). In general Zephyr devs prefer to use '#ifdef` when both are fine.